Src/Web/Web.Shared.Net/Implementation/WebEventSource.cs

```diff
@@ -270,7 +270,19 @@ public void WebRequestFilteredOutByRequestHandler(string appDomainName = "Incorr
                 30,
                 this.ApplicationName);
         }
-      
+
+        [Event(
+            31,
+            Message = "[msg=SyntheticUserAgentTelemetryInitializerRegularExpressionParsingException];[exception={0}];",
```

Comment:
Looks like a user actionable error. Let's add more user friendly message and add UserActionable keyword